package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Girls-n-a-Dude",
+  "version": "1.0.0",
+  "author": "",
+  "license": "MIT",
+  "scripts": {
+     "start": "node src/server.js",
+    "start:watch": "nodemon -q src/server.js"
+  },
+  "dependencies": {
+    "express": "^4.15.3",
+    "express-handlebars": "^3.0.0",
+    "serve-favicon": "^2.4.3"
+  },
+  "devDependencies": {
+    "istanbul": "^0.4.5",
+    "nodemon": "^1.11.0",
+    "shot": "^3.4.2",
+    "tap-spec": "^4.1.1",
+    "tape": "^4.8.0"
+  }
```

Comment:
@philawsophizing thanks for the comments. We are changing the devStart, and we are planning to use all of these dependencies, if time allows us :wink: